relay/abi.go

```diff
@@ -157,12 +158,62 @@ func generateRequestedReportFormMsg(event RequestedReportForm) (err error) {
 	return
 }
 
+func generatePostedKeyMsg(event PostedKey) (err error) {
+	log.Printf("Event details: %+v", event)
+	evmAddress = event.From.String()
+
+	relayedMsg = evmTypes.ExecuteMsg{
+		PostKey: &evmTypes.ExecuteMsgPostKey{
+			Key: event.Key,
+		},
+	}
+
+	// cost = int64(float64(q.GetCost(0, 0)) * 1.2)
+	return
+}
+
+func generateDeletedFileTreeMsg(event DeletedFileTree) (err error) {
+	log.Printf("Event details: %+v", event)
+	evmAddress = event.From.String()
+
+	h := sha256.New()
+	h.Write([]byte(event.Account))
+	accountHash := h.Sum(nil)
+
+	relayedMsg = evmTypes.ExecuteMsg{
+		DeleteFileTree: &evmTypes.ExecuteMsgDeleteFileTree{
+			HashPath: MerklePath(event.HashPath),
```

Comment:
See #22

Reply:
fixed (this is just based off the previous `delete-file-tree` branch)